fwup-ops.conf

```diff
@@ -86,7 +86,7 @@ task revert.a {
 
         # Switch over
         uboot_setenv(uboot-env, "nerves_fw_active", "a")
-        uboot_setenv(uboot-env, "nerves_fw_validated", "1")
+        uboot_setenv(uboot-env, "a.nerves_fw_validated", "1")
```

Comment:
I feel like splitting `nerves_fw_validated` into `a.` and `b.` is going to complicate the tasks when we add a non-upgradable `c` partition. We can talk about it.